api/swagger.yaml

```diff
@@ -8229,7 +8229,7 @@ paths:
 
             Available filters:
 
-            - `until=<duration>`: duration relative to daemon's time, during which build cache was not used, in Go's duration format (e.g., '24h')
+            - `until=<timestamp>` Prune build cache which wasn't used before this timestamp. The `<timestamp>` can be Unix timestamps, date formatted timestamps, or Go duration strings (e.g. `10m`, `1h30m`) computed relative to the daemon machine’s time.
```

Comment:
Need to have a closer look, but the description is a bit confusing; it's removing cache "older than" `until`, right?

Reply:
Yes, all cache that was last used before the `until` time is deleted.

Reply:
@dvdksn do you have good suggestions for the description? (I'm failing to come up with a good one right now 😓 😅)

Reply:
Something like "remove all cache older than `<timestamp>`" (with the existing note description of format for `<timestamp>`)?

Reply:
"Remove cache older than" sgtm!

I guess we could update container/prune to use the same language (remove instead of prune, in the description)